### PR TITLE
fix: fix dashboardsView cypress test flakiness

### LINKED ISSUE #1201: Flaky Test: can manage variable state with a lot of pointing and clicking
Flaky test:
- shared/dashboardsView.test.ts -- variable interactions -- can manage variable state with a lot of pointing and clicking 
- https://github.com/influxdata/ui/blob/master/cypress/e2e/shared/dashboardsView.test.ts#L288

Example failure:
- https://app.circleci.com/pipelines/github/influxdata/ui/3803/workflows/5077c585-d68d-4e15-9ac8-ea9aff2a050c/jobs/29854/artifacts
- https://app.circleci.com/pipelines/github/influxdata/monitor-ci/165/workflows/11b26c40-f099-4966-aa6c-8f725106ac31/jobs/1118/artifacts

Also this test is massive - look if it's possible to break it up into multiple tests

### PR DESCRIPTION
**Pull request by subirjolly**
Closes #1201

### CODE REVIEW

File: cypress/e2e/shared/dashboardsView.test.ts [hunk 1]
```diff
@@ -337,6 +337,12 @@ describe('Dashboard', () => {
                   })
                 cy.getByTestID('save-cell--button').click()
 
+                // Make sure typeAhead input box is rendered and is visible
+                cy.getByTestID(
+                  `variable-dropdown-input-typeAhead--${bucketVarName}`,
+                  {timeout: 10000}
```
Comment (genehynson) [inline cypress/e2e/shared/dashboardsView.test.ts:343]: I think this is the default timeout, right? 10 seconds

Comment (subirjolly) [reply cypress/e2e/shared/dashboardsView.test.ts:343]: that's correct

Comment (Asalem1) [reply cypress/e2e/shared/dashboardsView.test.ts:343]: wait, why are we manually setting a timeout?

**Review by Asalem1 — CHANGES_REQUESTED** [review]

File: cypress/e2e/shared/dashboardsView.test.ts (hunk 1 above)
Comment (subirjolly) [reply cypress/e2e/shared/dashboardsView.test.ts:343]: `defaultCommandTimeout` is 4s but somehow asserting should before it's visible is breaking

Comment (subirjolly) [reply cypress/e2e/shared/dashboardsView.test.ts:343]: Remove it?

**Comment by hercules-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 59 | :red_circle:
accessibility | 96 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 93 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?vars%5Blighthouse_score_branch%5D=1203)

**Review by genehynson — APPROVED** [review]
thanks!

**Review by Asalem1 — APPROVED** [review]

**Pull request merged**